systemd/ceph-radosgw-prestart.sh

```diff
@@ -51,35 +56,35 @@ if [ ! -f "${ceph_conf_file}" ] ; then
 fi
 
 longname=${PREFIX}${name}
-testname=$(${CEPHCONF} -c ${ceph_conf_file} --list-sections $PREFIX | grep $longname )
+testname=$(${CEPHCONF} --cluster ${cluster} -c ${ceph_conf_file} --list-sections $PREFIX | grep $longname )
 
 if [ -z "$testname"  ]; then
-  echo "error parsing '$name' : valid types are: $(echo $(${CEPHCONF} -c ${ceph_conf_file} --list-sections $PREFIX | sed s/$PREFIX//))"
+  echo "error parsing '$name' : valid types are: $(echo $(${CEPHCONF} --cluster ${cluster} -c ${ceph_conf_file} --list-sections $PREFIX | sed s/$PREFIX//))"
   exit 1
 fi
 
-auto_start=`${CEPHCONF} -c ${ceph_conf_file} -n $longname 'auto start'`
+auto_start=`${CEPHCONF} --cluster ${cluster} -c ${ceph_conf_file} -n $longname 'auto start'`
 if [ "$auto_start" = "no" ] || [ "$auto_start" = "false" ] || [ "$auto_start" = "0" ]; then
-  echo "ceph.conf:[$longname], says not to start."
+  echo "${ceph_conf_file}:[$longname], says not to start."
```

Comment:
Here, this might be quite long, especially since full path is given, could we go with something like '$(basename ${ceph_conf_file})', here?